demisto_sdk/tests/integration_tests/secrets_integration_test.py

```diff
@@ -25,23 +24,26 @@ def test_integration_secrets_incident_field_positive(mocker):
     - Ensure secrets validation passes.
     - Ensure success secrets validation message is printed.
     """
+    # Mocking the git functionality (Else it'll raise an error)
+    mock_git(mocker)
+    # Change working dir to repo
+    os.chdir(integration.repo_path)
     mocker.patch(
         "demisto_sdk.__main__.SecretsValidator.get_all_diff_text_files",
         return_value=[
-            join(DEMISTO_SDK_PATH,
-                 "tests/test_files/content_repo_example/Packs/FeedAzure/IncidentFields/incidentfield-city.json"
-                 )
+            integration.yml_path
         ]
     )
+    integration.write_yml({'valid yml file': 'really'})
```

Comment:
if I get/create an integration then I already should have basic valid integration yml. 
We need basic integration yml for later, for example validation flow.
When we do `integration_a = create_integration` it means it creates basic integration folder and required files in it.